doc/reference/alerttypes.rst

```diff
@@ -109,13 +109,6 @@ Tells us whether the load has passed a certain threshold.
 ------------------
 Tells us whether a link is up or down.
 
-.. list-table:: Alerts associated with linkState events
-   :widths: 25 75
-   :header-rows: 1
-
-   * - Alert type name
-     - Description
-
```

Comment:
As mentioned in other channels, `alerttypes.rst` is actually supposed to be auto-generated using the script found in `doc/ext/alerttypes.py`, which creates output based on the registered event and alert types found in the SQL database.  This is a manual operation that needs to be done each time someone changes event types, but I don't think we have documented this necessity very well?